packages/web/src/components/basic/TagCloud.js

```diff
@@ -204,15 +216,35 @@ class TagCloud extends Component {
 	static generateQueryOptions(props) {
 		const queryOptions = getQueryOptions(props);
 		queryOptions.size = 0;
-		queryOptions.aggs = {
-			[props.dataField]: {
-				terms: {
-					field: props.dataField,
-					size: props.size,
-					order: getAggsOrder(props.sortBy || 'asc'),
+		const { nestedField } = props;
```

Comment:
We should use `getAggsQuery(queryOptions, props)` util here. Check `SingleList` component for further implementation details.